backend/src/users/users.module.ts

```diff
@@ -13,6 +13,10 @@ import { User } from './entities/user.entity';
       debug: true,
       playground: true,
       autoSchemaFile: true,
+      cors: {
+        origin: 'http://localhost',
```

Comment:
We'll probably have problems with that later, but we also have a cors * in the websockets thing for the chat. I'll create and issue to check all that before evaluation

Reply:
Yes we will need to change every hardcoded localhost later. About CORS, even for development, we probably can't use * wildcard because it doesn't work when working with crendentials (cookies). [Relevant](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS#examples_of_access_control_scenarios). This is to prevent CSRF attack